mod.ts

```diff
@@ -1 +1,2 @@
 export * from "https://cdn.jsdelivr.net/gh/kakomimasu/server@v1.0.0-beta.8/client/client.ts";
+export { default as ApiClient } from "https://cdn.jsdelivr.net/gh/kakomimasu/server@v1.0.0-beta.8/client/client.ts";
```

Comment:
```suggestion
import { default as ApiClient } from "https://cdn.jsdelivr.net/gh/kakomimasu/server@v1.0.0-beta.8/client/client.ts";
export default ApiClient; 
```

こうしないとdefautl exportにならないんでした...